docs/index.md

```diff
@@ -102,6 +102,12 @@ connection "aws" {
   # If not set, the default AWS generated endpoint will be used.
   # Can also be set with the AWS_ENDPOINT_URL environment variable.
   #endpoint_url = "http://localhost:4566"
+  
+  # Set to `true` to force S3 requests to use path-style addressing,
```

Comment:
Please remember to make any similar changes from `aws.spc` here as well

Reply:
Done